src/main/java/business/representation/RequestListRepresentation.java

```diff
@@ -69,6 +70,8 @@
 
     private Long dataAttachmentCount;
 
+    private ContactData billingAddress;
```

Comment:
De code voor het ophalen van de aanvraag zit in de `RequestController`: https://github.com/thehyve/dntp-portal/blob/788e1869ff1df32d22c7c958fde716cd33299e76/src/main/resources/static/app/request/RequestController.js#L43